app/models/artefact.rb

```diff
@@ -344,7 +344,7 @@ def validate_redirect_url
 
   def valid_redirect_url_path?(target)
     URI.parse(target)
-    target.starts_with?("/") && target !~ %r{//} && target !~ %r{./\z}
+    target !~ %r{[^:]//}
```

Comment:
> Do you think it even needs a check? would we be expecting this to be validated by schemas anyway?

Not really, I'd be quite happy to remove it. I've put up a PR to that effect here https://github.com/alphagov/govuk_content_models/pull/433 .